src/custom/state/orders/updater.ts

```diff
@@ -21,6 +21,7 @@ import { stringToCurrency } from '../swap/extension'
 type OrderLogPopupMixData = OrderFulfillmentData & Pick<Log, 'transactionHash'> & Partial<Pick<Order, 'summary'>>
 
 const TradeEvent = GP_V2_SETTLEMENT_INTERFACE.getEvent('Trade')
+const DEFAULT_ORDER_DELAY = 20000 // 20 seconds
```

Comment:
consider moving to const

Reply:
> I'm good with this PR, but I believe we need something more robust aligned with the ideas in #213

I'll add a comment that this is a temporary fix